src/utils/camera-utils.ts

```
@@ -0,0 +1,83 @@
+          facingMode: 'environment'
+        }
+      })
+    } catch {
+      return null
+    }
+  }
+
+  static stopCameraStream(stream: MediaStream): void {
+    stream.getTracks().forEach(track => track.stop())
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)